event/register_oracle_event.go

```diff
@@ -0,0 +1,26 @@
+package event
+
+var _ Event = (*RegisterOracleEvent)(nil)
+
+type RegisterOracleEvent struct {
+	EventType           string
+	EventAttributeKey   string
+	EventAttributeValue string
+	EventHandler        func() error
+}
+
+func (e RegisterOracleEvent) GetEventType() string {
+	return e.EventType
+}
+
+func (e RegisterOracleEvent) GetEventAttributeKey() string {
+	return e.EventAttributeKey
+}
+
+func (e RegisterOracleEvent) GetEventAttributeValue() string {
+	return e.EventAttributeValue
+}
+
+func (e RegisterOracleEvent) GetEventHandler() func() error {
+	return e.EventHandler
+}
```

Comment:
```suggestion
type RegisterOracleEvent struct {}

func (e RegisterOracleEvent) GetEventType() string {
	return "message"
}

func (e RegisterOracleEvent) GetEventAttributeKey() string {
	return "action"
}

func (e RegisterOracleEvent) GetEventAttributeValue() string {
	return "RegisterOracle"
}

func (e RegisterOracleEvent) EventHandler() error {
	// TODO: Executing Voting Tx
	fmt.Println("RegisterOracle Event Handler")
	return nil
}
```
You can implement this `RegisterOracleEvent` struct statically because the logic of these 4 functions won't be changed dynamically.

Reply:
Oh, I got it. It makes sense. Thank you!